tests/test_admin.py

```diff
@@ -667,6 +667,64 @@ def test_menuitem_changelist_should_have_get_url_column(self):
         self.assertEqual(len(ma.get_list_display(mock_request)), 4)
         self.assertIn("get_object_url", ma.get_list_display(mock_request))
 
+    def test_menuitem_add_view_redirects_on_save_Continue(self):
+        """
+        redirect to edit url when Save and continue clicked
```

Comment:
This doesn't tell me why we are testing this, something like: When a MenuIem object is created we check that the correct url is generated and a message is shown to the user. 

No message here though?